tests/UnitTests/CoreGraphics.drawing/data/.keep

```diff
@@ -0,0 +1 @@
+
```

Comment:
Git does not track directories, it tracks files. This is an empty file intended to keep the data directory around.